Makefile.Common

```diff
@@ -36,7 +36,8 @@ all-srcs:
 .DEFAULT_GOAL := common
 
 .PHONY: common
-common: checklicense checkdoc impi lint misspell
+#common: checklicense checkdoc impi lint misspell
+common: checklicense checkdoc impi misspell
```

Comment:
Any reason for removing lint ?

Reply:
It is disabled since it fails it also disabled for now in _circleci_ see https://github.com/open-telemetry/opentelemetry-collector/issues/2789   - it should not be part of changes.